requirements.txt

```diff
@@ -1,6 +1,21 @@
-Authlib==1.2.0
```

Comment:
Good catch - this must of been added by accident.  setup.cfg are things the library needs to function while requirements.txt is for development dependencies...   With that said I see Jinja2 and openpyxl that should be moved to setup.cfg - otherwise they won't be installed by the third party app...   (Unless it does and my assumption is wrong - or the intent is to have consumers add them manually - in which case there should be a note in the readme about that)

Reply:
Not every consumer will need every package. The goal is to have users install them manually. The excel package will raise a custom exception if it's imported without the package installed. In the case of jinja, I would hope it's obvious enough, and consumers can probably tinker with versions without this package breaking

Reply:
Also, at this point I've just added the jinja package as an unused dev-dependency. I could remove it but I figure jinja helpers will be probably be added soon, or at least other utils' tests will depend on templating. Nothing in the published package currently depends on jinja in anyway, so I think the custom exception is enough for the excel package. 

Documenting the excel stuff in the README would be nice but there are so many micro-utils I don't really think it's worth it, reading the code will be easy enough.

I think we can revisit the idea of optional dependencies, either by using the optional deps in the `setup.cfg`, raising custom exceptions or just documenting stuff. I know modeltranslation isn't really being used by any projects anymore, I think the catalog has ditched it